template-react-typescript/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Vite App</title>
+</head>
+<body>
+  <div id="root"></div>
+  <script type="module" src="./src/main.tsx"></script>
```

Comment:
```suggestion
  <script type="module" src="/src/main.tsx"></script>
```

This is necessary for SPA with history fallback